common/flatpak-run.c

```diff
@@ -2400,6 +2400,8 @@ add_icon_path_args (FlatpakBwrap *bwrap)
                               "--ro-bind", flatpak_file_get_path_cached (user_icons), "/run/host/user-share/icons",
                               NULL);
     }
+  /* We want to set XCURSOR_PATH so that apps know where to look for cursors. Otherwise, they use a default cursor that doesn't fit in well. */
+  flatpak_bwrap_set_env (bwrap, "XCURSOR_PATH", "/run/host/user-share/icons:/run/host/share/icons", TRUE);
```

Comment:
This isn't the same as the search path used in the fdo SDK, and in particular doesn't include `/app/share` or `/usr/share`. Should it?

Reply:
I don't think so. The paths that a runtime includes for searching could be specific to that runtime, it's entirely possible that a different runtime uses different paths and the one thing they have in common is this path. 

It's unlikely, but possible. I don't think flatpak should assume every runtime it uses is based on the fd.o runtime.

Reply:
This is true but then the runtime needs to set this env var itself anyway.

Reply:
Could the runtime not build off of the variable set by Flatpak? 

Tbh what it does already fine, and it basically does that. This PR is mainly to fix an issue when the variable isn't set automatically (as seems to be the case for Chromium and derivs).

Reply:
Not really, env vars are static data at build time. 

This patch isn’t enough and the runtime needs changes anyway.

Reply:
> This patch isn’t enough and the runtime needs changes anyway.

Oh? Last I checked, the runtime already set this variable in a similar fashion, just with some more paths. The only reason it breaks on Chromium is because it ships its own Wayland libraries apparently.